tardis/visualization/plot_util.py

```diff
@@ -248,7 +261,12 @@ def process_line_interactions(packet_data, lines_df):
         )
 
 
-def extract_and_process_packet_data_hdf(hdf, packets_mode):
+def extract_and_process_packet_data_hdf(
+    hdf,
+    packets_mode,
+    lines_hdf_path="/simulation/plasma_solver/lines",
```

Comment:
Shouldnt we pass plasma here instead of lines_hdf just for consistency with simulation method?